blocks/categories.js

```diff
@@ -0,0 +1,18 @@
+
+/**
+ * Block categories
+ *
+ * @var {Array} categories
+ */
+const categories = [
+	{ id: 'common', label: 'Common' }
```

Comment:
Instead of `id` and `label`, I'm thinking we should use the same convention as blocks itself, which have a `slug` and a `title`.